src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1132,6 +1185,17 @@ end
 theorem lsub_const {ι} [hι : nonempty ι] (o : ordinal) : lsub (λ _ : ι, o) = o + 1 :=
 sup_const o.succ
 
+theorem lsub_le_of_range_subset {ι ι'} {f : ι → ordinal} {g : ι' → ordinal}
+  (h : set.range f ⊆ set.range g) : lsub f ≤ lsub g :=
+sup_le_of_range_subset begin
+  convert set.image_subset _ h;
+  apply set.range_comp
+end
```

Comment:
I think it's okay to put this on one line.
```suggestion
sup_le_of_range_subset (by convert set.image_subset _ h; apply set.range_comp)
```

Reply:
please change `;` by `,` unless the first tactic actually creates more than 1 goal (I didn't check if it does).

Reply:
It creates two goals, actually.